src/makeCalculator.js

```diff
@@ -4,7 +4,44 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  let result = 0;
```

Comment:
Nicely done!)🔥 You can also do this task by creating just one object and including result as a key of the object. I like your version, it is even shorter)